src/style.css

```
@@ -0,0 +1,60 @@
+body {
+  background: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+}
+
+nav {
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: flex-end;
+  margin: 0 auto;
+  background: #fff;
+  padding: 0 50px;
+}
+
+.logo-box {
+  flex-grow: 1;
+  position: relative;
+  bottom: 10px;
+}
+
+.nav_link_container {
+  padding: 22.5px 0;
```

```suggestion
  padding: 22px 0;
```
Не используй дробные значения
Вообще тут можно просто задавать высоту, равную высоте хедера
https://codepen.io/adskiy/pen/gOWGeLV
Вот пример